README.md

```diff
@@ -1,28 +1,41 @@
-# Metal³ Documentation
+<p align="center"><img alt="Metal³" src="./images/metal3.png" /></p>
 
-![Metal³ Logo](images/metal3.png)
+# Metal³
+
+## Useful links
+
+* [Quick start](URL)
+* [Demos](https://www.youtube.com/watch?v=VFbIHc3NbJo&list=PL2h5ikWC8viKmhbXHo1epPelGdCkVlF16&ab_channel=Metal3)
+* [Blog posts](https://metal3.io/blog/index.html)
+* [FAQ](https://metal3.io/blog/index.html)
+
+## What is the Metal³
 
 The Metal³ project (pronounced: Metal Kubed) exists to provide components that
 allow you to do bare metal host management for Kubernetes.  Metal³ works as a
 Kubernetes application, meaning it runs on Kubernetes and is managed through
 Kubernetes interfaces.
 
-## Social Media
-
-* [twitter.com/metal3_io](https://twitter.com/metal3_io)
-
-## Project Discussion
-
-* [Metal³ Development Mailing List](https://groups.google.com/forum/#!forum/metal3-dev)
-* [#cluster-api-baremetal](https://kubernetes.slack.com/messages/CHD49TLE7)
-  on Kubernetes Slack
-* Bi-Weekly Community Meetings every alternate Wednesday at 13:00 UTC
-  on [Zoom](https://zoom.us/j/97255696401?pwd=ZlJMckNFLzdxMDNZN2xvTW5oa2lCZz09)
-* Previous meetings: \[ [notes][notes] | [recordings][recordings] \]
-
-## More Documentation
-
-For more documentation, please see <http://metal3.io/documentation.html>
-
-[notes]: https://docs.google.com/document/d/1d7jqIgmKHvOdcEmE2v72WDZo9kz7WwhuslDOili25Ls/edit
-[recordings]: https://www.youtube.com/playlist?list=PL2h5ikWC8viJY4SNeOpCKTyERToTbJJJA
+## Community
+
+Metal³ is constantly growing and we are very happy with every single contribution
```

Comment:
I think I understand the intent, but I don't agree with "we are very happy with every single contribution." We need to balance encouraging contributions from newcomers and implying that we'll take whatever code is dumped on us and maintain it indefinitely. Maybe say "we would be happy to collaborate with you"?

Reply:
Fixed, thanks.